pycloudlib/instance.py

```diff
@@ -398,28 +408,27 @@ def _tmpfile(self):
         self._tmp_count += 1
         return path
 
-    def _wait_for_execute(self):
-        """Wait until we can execute a command in the instance."""
-        self._log.debug("_wait_for_execute to complete")
-        test_instance_command = "whoami"
+    def _wait_for_boot_id(self):
```

Comment:
As it is implemented now I think this should be called `_wait_for_new_boot_id()`, but see my other comment on issues with always checking for new boot_id on `wait()`.